src/hoc/ClientSideStateUpdate.tsx

```diff
@@ -87,6 +90,10 @@ export const ClientSideStateUpdate = ({
       const updatedTasks = tasksInStore.map((t) => (t.id === task.id ? task : t))
       store.dispatch(setTasks(updatedTasks))
     } //for updating a task in store with respect to task response from db in task details page
+
+    if (activities) {
+      store.dispatch(setActivities(activities))
```

Comment:
I don't see activities being passed as a prop to `ClientSideStateUpdate` anywhere(?)

Reply:
it is not needed anymore. will remove.